source/Octopus.Cli/Commands/Releases/ReleasePlanBuilder.cs

```diff
@@ -163,7 +163,7 @@ async Task<ReleasePlan> BuildReleaseFromDatabase(IOctopusAsyncRepository reposit
 
                     if (latestPackage == null)
                     {
-                        commandOutputProvider.Error(
+                        commandOutputProvider.Information(
                             "Could not find any packages with ID '{PackageId:l}' in the feed '{FeedUri:l}'",
```

Comment:
I think the text in this message is misleading. It isn't that no packages could be found, it is that none matching the filter criteria for the channel rule could be found. As far as understanding what is happening, it may read better as something like "Could not find any packages with ID '{PackageId:l}' **that match the channel filter,** in the feed '{FeedUri:l}'"

Reply:
yep, agreed